rebasehelper/specfile.py

```diff
@@ -125,11 +127,35 @@ def __init__(self, path, sources_location='', download=True):
         rpm.addMacro("_sourcedir", self.sources_location)
         self._read_spec_content()
         # Load rpm information
-        self.spc = rpm.spec(self.path)
-        self.patches = self._get_initial_patches_list()
```

Comment:
```_update_data()``` is called in ```__init__()``` of SpefFile on line 131 (2 lines below your comment). Those lines were redundant as they are included in ```_update_data()``` and were called twice in case of creation of class object.